.github/workflows/nodejs-macos.yml

```
@@ -22,3 +22,14 @@ jobs:
         uses: ./.github/actions/prepare
       - name: Build & Test
         uses: ./.github/actions/test
+  re-run:
```

```suggestion
  rerun:
```

(to match file/job names)